hdr.go

```diff
@@ -292,21 +292,34 @@ func (h *Histogram) Equals(other *Histogram) bool {
 // Export returns a snapshot view of the Histogram. This can be later passed to
 // Import to construct a new Histogram with the same state.
 func (h *Histogram) Export() *Snapshot {
+	counts := make([]int64, 0, len(h.counts))
```

Comment:
@tylertreat Is this here correct to increase the size of the counts as need fit and not make it as big as `h.counts` is already?

I will add a test to be sure I guess :-)